native/cocos/scene/Model.h

```diff
@@ -188,6 +193,7 @@ class Model : public RefCounted {
 
     Type _type{Type::DEFAULT};
     bool _localDataUpdated{false};
+    bool _needUpdateOctree{true};
```

Comment:
When set `_needUpdateOctree ` to true?

Reply:
when octree is just updated

Reply:
Got it. I suggest to change `_needUpdateOctree ` to `_isWorldBoundsDirty`.

Reply:
1